kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -59,6 +60,13 @@
   private static final String TOPICS_DOC =
       "A list of Kafka topics to read from";
 
+  public static final String ENABLE_BATCH_CONFIG =                         "enableBatchLoad";
+  public static final ConfigDef.Type ENABLE_BATCH_TYPE =                   ConfigDef.Type.LIST;
+  public static final List<String> ENABLE_BATCH_DEFAULT =                  Collections.emptyList();
+  public static final ConfigDef.Importance ENABLE_BATCH_IMPORTANCE =       ConfigDef.Importance.LOW;
+  public static final String ENABLE_BATCH_DOC =
+      "A list of Kafka topics to batch load through GCS";
```

Comment:
We should mention we expect this to be a sub-list of the full topics list.
Maybe something like
"The sublist of topics to be batch loaded through GCS"?